CMakeLists.txt

```diff
@@ -50,8 +50,13 @@ message(STATUS "Internal miniwin:       ${ISLE_MINIWIN}")
 message(STATUS "Isle debugging:         ${ISLE_DEBUG}")
 message(STATUS "Compile shaders:        ${ISLE_COMPILE_SHADERS}")
 
+
+
 if (DOWNLOAD_DEPENDENCIES)
     # FetchContent downloads and configures dependencies
+
+    message(STATUS "Fetching SDL3 and iniparser. This might take a while...")
+
     include(FetchContent)
```

Comment:
Reduce whitespace.
```suggestion
message(STATUS "Compile shaders:        ${ISLE_COMPILE_SHADERS}")

if (DOWNLOAD_DEPENDENCIES)
    # FetchContent downloads and configures dependencies
    message(STATUS "Fetching SDL3 and iniparser. This might take a while...")
    include(FetchContent)
```